broker-core/pom.xml

```diff
@@ -142,6 +153,24 @@
 			<artifactId>elasticsearch</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.elasticsearch.client</groupId>
+			<artifactId>transport</artifactId>
+			<version>${elasticsearch-client-transport.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+	    	<groupId>org.apache.logging.log4j</groupId>
```

Comment:
About formatting issues in general I have no idea about these. I have the style correctly loaded in my Eclipse IDE so I supposed the code was formatted properly.